src/ds/ds-motion-common.cpp

```diff
@@ -507,10 +507,12 @@ namespace librealsense
         }
         catch (...) {}
 
+        bool high_accuracy =  _fw_version >= firmware_version( 5, 16, 0, 0 ) ? true : false ;    
```

Comment:
Minor but no need for true : false
`bool high_accuracy =_fw_version >= firmware_version( 5, 16, 0, 0 );`

Reply:
Done